python/ray/experimental/sgd/tests/tf_helper.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import tensorflow as tf
+
+NUM_TRAIN_SAMPLES = 60000
```

Comment:
can we make this 512?

Reply:
shuffle(NUM_TRAIN_SAMPLES) is used to shuffle whole data. 
Are you meaning we should only consider 512 datapoints from mnist dataset?